inference-engine/tests/functional/plugin/cpu/single_layer_tests/gather_elements.cpp

```diff
@@ -2,88 +2,142 @@
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (int i = 0; i < funcInputs.size(); ++i) {
+            const auto& funcInput = funcInputs[i];
+            ov::runtime::Tensor tensor;
+
+            tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i], 15, 0, 32768);
+
+            inputs.insert({funcInput.get_node_shared_ptr(), tensor});
+        }
     }
 
 protected:
     void SetUp() override {
-        InferenceEngine::SizeVector dataShape, indicesShape;
-        InferenceEngine::Precision dPrecision, iPrecision;
+        std::vector<InputShape> shapes;
+        ElementType dPrecision, iPrecision;
         int axis;
-
         GatherElementsParams basicParamsSet;
         CPUSpecificParams cpuParams;
         std::tie(basicParamsSet, cpuParams) = this->GetParam();
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
 
-        std::tie(dataShape, indicesShape, axis, dPrecision, iPrecision, targetDevice) = basicParamsSet;
-        selectedType = std::string("ref_any_") + dPrecision.name();
+        std::tie(shapes, axis, dPrecision, iPrecision, targetDevice) = basicParamsSet;
+        selectedType = std::string("ref_any_") + ov::element::Type(dPrecision).get_type_name();
+        init_input_shapes(shapes);
 
-        auto ngDPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(dPrecision);
-        auto ngIPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(iPrecision);
+        const ngraph::ParameterVector params = {
+            std::make_shared<ngraph::opset1::Parameter>(dPrecision, inputDynamicShapes[0]),
+            std::make_shared<ngraph::opset1::Parameter>(iPrecision, inputDynamicShapes[1]),
+        };
 
-        auto params = ngraph::builder::makeParams(ngDPrc, {dataShape});
-        auto activation = ngraph::builder::makeGatherElements(params[0], indicesShape, ngIPrc, axis);
-        activation->get_rt_info() = getCPUInfo();
-        function = std::make_shared<ngraph::Function>(ngraph::NodeVector{activation}, params, "GatherElements");
+        auto gather = std::make_shared<ngraph::op::v6::GatherElements>(
+            params[0], params[1], axis);
+        gather->get_rt_info() = getCPUInfo();
+        ngraph::ResultVector results{std::make_shared<ngraph::opset1::Result>(gather)};
+        function = std::make_shared<ngraph::Function>(results, params, "GatherElements");
     }
 };
 
 TEST_P(GatherElementsCPUTest, CompareWithRefs) {
-    SKIP_IF_CURRENT_TEST_IS_DISABLED()
-
-    Run();
-    CheckPluginRelatedResults(executableNetwork, "GatherElements");
+    run();
 }
 
-
 namespace {
 std::vector<CPUSpecificParams> cpuParams_4D = {
         CPUSpecificParams({nchw}, {nchw}, {}, {})
 };
 
+const std::vector<std::vector<InputShape>> inDynamicShapeParams = {
+    {{{ngraph::Dimension::dynamic(), ngraph::Dimension::dynamic(), ngraph::Dimension::dynamic(), ngraph::Dimension::dynamic()},
```

Comment:
you can use `-1` instead `ngraph::Dimension::dynamic()`
and `{1, 10}` instead `ngraph::Dimension(1, 10)`

Reply:
Done.